config/routes.rb

```diff
@@ -12,6 +12,10 @@
     resources :objectives, only: %i[new create]
     resources :invitations, only: %i[index new create destroy], controller: 'account_invitations'
   end
+
+  get 'account_invitation/:invitation_id', to: 'account_invitations#accept_invitation', as: :visit_by_account_invitation
+  post 'account_invitation/:invitation_id', to: 'account_invitations#accept', as: :accept_account_invitation
```

Comment:
can we put them into the block that defines invitations as members (https://guides.rubyonrails.org/routing.html#adding-member-routes)?